### ISSUE TITLE
[WP 6.2] Hybrid themes: wide and full width settings are lost from the toolbar

### ISSUE THREAD

**Issue by ya123123** [WordPress/gutenberg#47857]
### Description

When I use hybrid theme in WordPress 6.2 beta1 nightly, wide and full width settings are lost from the toolbar.

Also, the wide/full-width cover block becomes the content width.

### Step-by-step reproduction instructions

Tested with WordPress 6.2 beta1 nightly without Gutenberg plugin.

1. Use a classic theme without theme.json. I use TT1 (Twenty Twenty-one).
2. Add blocks to the post and set its width to wide or full.
3. Add TT3's theme.json to TT1 theme folder. TT1 is now a hybrid theme.
4. Open the post editor again, wide and full width settings are lost from the toolbar.

### Screenshots, screen recording, code snippet

| - | TT1 without theme.json  | TT1 with theme.json |
| - | ------------- | ------------- |
| Cover | ![cover01](https://user-images.githubusercontent.com/114547428/217434908-6ac1c7e5-63ba-47e3-986a-c61b5e253fd6.jpg) | ![cover02](https://user-images.githubusercontent.com/114547428/217434938-cc024424-5367-4a96-8bff-7c5e331023a9.png) |
| Image | ![image01](https://user-images.githubusercontent.com/114547428/217435046-9cc535ab-021d-41a8-9e77-cf1ae6847228.jpg) | ![image02](https://user-images.githubusercontent.com/114547428/217435064-5f707cf6-2759-4052-b87f-7eec2ca701f3.jpg) |
|  |  |

### Environment info

WP6.2-beta1-55293 (latest nightly via Beta Tester plugin), instawp install
TT1 and theme.json of TT3 included in nightly.
Tested without Gutenberg plugin

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by carolinan** [comment]
I can reproduce this.
6.2-beta1-55293 via beta tester plugin, with or without Gutenberg trunk active.
Mac Os, PHP 8.1.9, Chrome

**Comment by Mamaduka** [comment]
I can also reproduce the issue. Alignments are available in WP 6.1.x but not when using the latest Gutenberg or 6.2-beta1.

I added the following `theme.json` file to TT1 for testing.

```json
{
	"$schema": "https://schemas.wp.org/trunk/theme.json",
	"version": 2,
	"settings": {
		"layout": {
			"contentSize": "800px",
			"wideSize": "1000px"
		}
	}
}
```

**Comment by ya123123** [comment]
Alighments now work in 6.2-beta2-55343. Thank you!
